docs/pages/tutorial/image-picker.md

```diff
@@ -34,7 +34,6 @@ With the library installed in our project, we can now actually use it.
 
 <!-- prettier-ignore -->
 ```js
-import React from 'react';
```

Comment:
Please revert this line.

Reply:
Reverted in the new commit. For consistency with the previous examples I agree it is good to keep it, but it seems that it is not used or included in the create-expo-app before that.